test/suite/test_tiered02.py

```diff
@@ -154,10 +150,6 @@ def test_tiered(self):
         self.progress('reopen_conn')
         self.reopen_conn()
 
-        # FIXME-WT-7589 This test works up to this point, then runs into trouble.
```

Comment:
Note, the ticket referenced in this comment (WT-7589) was previously closed to fix a bunch of earlier problems with this test.  When I did that, I moved the `if True: return` to later in the test, but I should have opened another ticket and updated the comment.  Anyway, it's fixed by this branch.